cypress/integration/home_page_spec.js

```diff
@@ -28,23 +14,16 @@ describe('The Login Page', function () {
     // UI should reflect this user being logged in
     cy.get('#players').should('contain', 'jane');
     });
-})
+});
+
 describe('Username already exists', function() {
-    afterEach(() => {
-        cy.request({
-            url: '/api/reset-server',
-            method: 'GET'
-        });
-        cy.visit('/');
-        });
     it('returns error', function () {
-        const username = 'Jane'
-        cy.request('POST', '/auth/login', { username: 'Jane' });
-        cy.get('input').type(username);
-        cy.get('button').click();
+        cy.request('POST', '/auth/login', { username });
```

Comment:
does this need the curly braces around it?

Reply:
Think so cause I want to send an object containing username? If I didn't have curly braces it would just send a string

Reply:
Oh, right.. I didn't know, was just curious